framework/src/outputs/Checkpoint.C

```diff
@@ -110,6 +112,13 @@ Checkpoint::shouldOutput(const ExecFlagType & type)
   // as the autosave and that checkpoint isn't on its interval, then output.
   if (_is_autosave == SYSTEM_AUTOSAVE || (_is_autosave == MODIFIED_EXISTING && !shouldOutput))
   {
+    auto curr_time = std::chrono::steady_clock::now();
+    auto elapsed = std::chrono::duration_cast<std::chrono::minutes>(curr_time - start_time);
+    if (elapsed >= std::chrono::minutes(10))
```

Comment:
Must this be hard-coded?  I've used a lot of systems where a save every 10 minutes would have felt inefficient/paranoid, but a save every hour would have been reassuring.

Reply:
No , it doesnt have to be. Changed it.